source/install.txt

```diff
@@ -1,177 +1,190 @@
-===============
-Install Compass
-===============
+============================
+Download and Install Compass
+============================
 
 .. default-domain:: mongodb
 
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: singlecol
+.. include:: /includes/styles/corrections.rst
 
-Compatibility
--------------
+.. tabs-pillstrip:: platforms
 
-|compass| works with MongoDB Server version 3.0 and higher, both
-Enterprise and Community editions. It runs on 64-bit versions
-of Mac OS X, Windows, Ubuntu 14.04+ and RHEL 7+.
+Software Requirements
+---------------------
 
-Software requirements to run |compass-short| are listed below:
+|compass-short| requires:
 
-.. include:: /includes/fact-compass-platforms.rst
+.. tabs-platforms::
 
-Installation
-------------
+   tabs:
+     - id: macos
+       content: |
 
-.. note::
+         - 64-bit version of macOS 10.10 or later
+         - MongoDB 3.0 or later.
 
-   As of MongoDB Server version 3.6, MongoDB packages a
-   platform-specific |compass-short| installation script with
-   `MongoDB Server downloads
-   <https://www.mongodb.com/download-center/enterprise>`_. For details,
-   see the :manual:`install_compass script
-   </reference/program/install_compass/>` documentation.
+     - id: windows
+       content: |
 
-.. tabs::
+         - 64-bit version of Microsoft Windows 7 or later
+         - MongoDB 3.0 or later.
+         - `Microsoft .NET Framework version 4.5 or later <https://www.microsoft.com/en-us/download/details.aspx?id=30653>`__.
+           
+           The |compass-short| installer prompts you to install the
+           minimum required version of the .NET framework if it is not
+           already installed on your system.
 
-   tabs:
-     - id: osx
-       name: OS X
+     - id: debian
        content: |
 
-         Open the {+download-page+} and download the latest version of
-         |compass| for OS X. The package downloads as a ``.dmg`` disk
-         image. Once you have downloaded |compass-short|, click on the
-         ``.dmg`` file to open the disk image within the OS X Finder.
-         Drag the |compass| application to your
-         :guilabel:`Applications` folder and unmount the disk image.
+         - 64-bit version of Ubuntu 14.04 or later
+         - MongoDB 3.0 or later.
 
-         .. figure:: /images/compass/osx-install-dialog.png
-            :figwidth: 600px
+     - id: rhel
+       content: |
 
-         Launch |compass-short| by clicking on it from within the
-         :guilabel:`Applications` folder. Upon launch you will see the
-         following dialog:
+         - 64-bit version of RHEL 7+ or later
```

Comment:
Add period at the end, since other bullet(s) has / have period(s) at the end.

Reply:
Agreed. Fixed.